app/views/unattended/provisioning_templates/finish/windows_default_finish.erb

```diff
@@ -14,12 +14,17 @@ params:
 - windowsLicenseOwner: Company, INC # Legal owner of the Windows license key
 - localAdminAccountDisabled: false
 - ntpServer: time.windows.com,other.time.server
-- domainAdminAccount: administrator@domain.com # use this account to join domain
+- domainAdminAccount: joinuser@domain.com # please do not use the domain administrator
```

Comment:
nit: If it's not he admin account, should this not be reflected in the the parameterName? Currently it says domainAdminAccount, perhaps domainJoinAccount?

Reply:
yeah thats true, but i wanted to leave the parameters the same for backwards compatibility.
also the username is used in the puppet section and i have no way to test the puppet installation..

Reply:
Okay, makes sense to me :-)